### PR TITLE
test/system: podman exec with short lived commands

### PR DESCRIPTION
**Pull request by chuanchang**
This test case is used for covering rhbz#1972311.

Signed-off-by: Alex Jia <ajia@redhat.com>

### CODE REVIEW

File: test/system/075-exec.bats [hunk 1]
```diff
@@ -129,4 +129,20 @@ load helpers
     run_podman rm -t 0 -f $cid
 }
 
+# rhbz#1972311 : podman exec -ti prints a resizing exec error when run
+# with short lived commands
+@test "podman exec with short lived commands" {
+    run_podman run -td $IMAGE sleep 5
+    cid=$output
+
+    run_podman exec -til ls /
```
Comment (vrothberg) [inline test/system/075-exec.bats:138]: Could you assign a (random) name to the container instead of `-l`?

**Review by Luap99 — CHANGES_REQUESTED** [review]
We already have a test for this, however it is disabled because it flakes to much.
https://github.com/containers/podman/blob/d6ef903164512e6fa7903bb0b85a1852117df37b/test/system/450-interactive.bats#L66-L73
Also see this issue https://github.com/containers/podman/issues/10710 for more context

**Comment by edsantiago** [comment]
What Paul said.

**Comment by rhatdan** [comment]
It seems that this PR was rejected, because flaky tests.  Closing.  Reopen if I am mistaken.

**Pull request closed without merging**